src/librustdoc/html/static/main.js

```diff
@@ -1221,7 +1221,7 @@ function getSearchElement() {
                     // then an exact path match
                     path.indexOf(keys[i]) > -1 ||
                     // next if there is a parent, check for exact parent match
-                    (parent !== undefined &&
+                    (parent !== undefined && parent.name !== undefined &&
```

Comment:
What does this commit have to do with "less minification" ?

Reply:
A little bug I encountered. I have put it into its own commit but maybe have been better into another PR, sorry...

Reply:
I guess it's okay here.